packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { InteractionModeEngine } from "../common/interactionMode";
+
+export const FOCUS_DISABLED_CLASS = "pt-focus-disabled";
```

Comment:
this should probably live in the `Classes` export... but unfortunately that would be a small breaking change.